src/Orleans.Serialization/Codecs/GuidCodec.cs

```diff
@@ -72,7 +72,17 @@ public static Guid ReadValue<TInput>(ref Reader<TInput> reader, Field field)
         {
             if (BitConverter.IsLittleEndian)
```

Comment:
This block could have been made conditional on NET7+ and the fallback would work on any other platform.